docs/design/002-airdrop.md

```diff
@@ -0,0 +1,100 @@
+# Design Doc 002: Airdrop Module
+
+## Status
+
+Draft
+
+## Abstract
+
+For the Ojo Network's Airdrop, we intend to require that users are only able to claim the other half of their tokens by staking their entire first half with a validator.
+
+## Context
+
+We want to incentivize airdrop recipients to stake their tokens. An effective way to do this is by rewarding users with the other half of their airdrop allocation if they stake to a validator.
+
+We also need to ensure that we're able to create vesting accounts for the initial airdrop amount, and controlling how long afterwards it will take to unlock. An issue here is that vesting accounts in cosmos can only be created for accounts which do not yet exist.
+
+## Specification
+
+### Msgs
+
+- `CreateAirdropAccount(address, tokensToReceive, amountExpectedToDelegate, vestingLength)` - Create an airdrop account which will unlock into a vesting account, if all the tokens in amountExpectedToDelegate are delegated. This transaction can only occur at genesis.
```

Comment:
Is there a benefit to having `amountExpectedToDelegate` as an extra function parameter rather than having a module parameter for percentage of `tokensToReceive` needed to be delegated or just flat out multiplying `tokensToReceive` by 0.5 since this only occurs in genesis.

Reply:
Fair yeah - we can have something like a const `airdropFactor`